src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationLineWrappedRecordDeclaration.java

```diff
@@ -54,8 +54,8 @@ private static String method(
                                                                                   //indent:82 exp:82
 interface SimpleInterface2 {                                                   //indent:0 exp:4 warn
 default                                                                        //indent:0 exp:8 warn
-void                                                                           //indent:0 exp:4 warn
-method()                                                                       //indent:0 exp:4 warn
+void                                                                          //indent:0 exp:12 warn
+method()                                                                      //indent:0 exp:12 warn
```

Comment:
Why expected indent is 12 here?

Reply:
"class InputIndentationLineWrappedRecordDeclaration" should have indentation 0, "interface SimpleInterface2" should have indentation 4 (basicOffset), "default" is beginning of method declaration and should have indentation 8 (basicOffset) and "void" and "method()" are line wrapped parts of method declaration and should have indentation 12 (lineWrappingIndentation)

Reply:
sounds good